pkg/workflows/create.go

```diff
@@ -68,6 +68,7 @@ func (c *Create) Run(ctx context.Context, clusterSpec *cluster.Spec, validator i
 	}
 
 	if packagesLocation != "" {
+		curatedpackages.PrintLicense()
```

Comment:
What is the purpose of printing out the license all the time?

Reply:
It is a business requirement to print out the license.